env/env.sh

```diff
@@ -1,4 +1,3 @@
 DOCKER_SOCKET=/var/run/docker.sock
-MOUNT=~/.developer-environment
```

Comment:
leave this in. i think we'll want to use the mount space eventually. when we know what config we want to persist. plus its used in the dev-env script still for dir creation

Reply:
Sorted